debian/changelog

```diff
@@ -1,3 +1,14 @@
+libfm-qt (1.1.0-0ubuntu7~ppa2) jammy; urgency=medium
+
+  * Backport to Jammy.
+  * Updated Lubuntu Developers copyright.
+  * Updated symbols file
+  * Removed Repository-Browser field from debian/upstream/metadata.
+  * Removed "-Wl,--as-needed" linker flag.
+  * Bumped Standards-Version (no changes necessary).
+
+ -- Aaron Rainbolt <arraybolt3@gmail.com>  Thu, 23 Jun 2022 21:47:05 -0500
```

Comment:
Perhaps bump the date while you're at it?

Reply:
Done.